### ISSUE TITLE
`loraplus_lr_ratio: 1` doesn't match lora behavior

### ISSUE THREAD

**Issue by ccdv-ai** [axolotl-ai-cloud/axolotl#2065]
### Please check that this issue hasn't been reported before.

- [X] I searched previous [Bug Reports](https://github.com/axolotl-ai-cloud/axolotl/labels/bug) didn't find any similar reports.

### Expected Behavior

When setting `loraplus_lr_ratio: 1` behavior should be similar to vanilla Lora, however this is not the case and loss values are significantly different.

### Current behaviour

`loraplus_lr_ratio: 1` doesn't match lora behavior.
Loss is higher, same behavior for different ratio values.

### Steps to reproduce

Running training with and without `loraplus_lr_ratio: 1`
lora config used:
```yaml
sample_packing: true

adapter: lora
loraplus_lr_ratio: 1
lora_r: 512
lora_alpha: 512
lora_dropout: 0.00
peft_use_rslora: false
lora_target_linear: true

gradient_accumulation_steps: 2
micro_batch_size: 4
num_epochs: 2
optimizer: adamw_torch
lr_scheduler: cosine
cosine_min_lr_ratio: 0.2
learning_rate: 0.00005
max_grad_norm: 1
```

### Which Operating Systems are you using?

- [X] Linux
- [ ] macOS
- [ ] Windows

### Python Version

3.10

### axolotl branch-commit

0.5.0

### Acknowledgements

- [X] My issue title is concise, descriptive, and in title casing.
- [X] I have searched the existing issues to make sure this bug has not been reported yet.
- [X] I am using the latest version of axolotl.
- [X] I have provided enough information for the maintainers to reproduce and diagnose the issue.

**Comment by ccdv-ai** [comment]
Looks like it is related to deepspeed.
Adding this for stage 1 fix it for me. Closing.
```json
"optimizer": {
        "type": "AdamW",
        "params": {
            "lr": "auto",
            "weight_decay": "auto",
            "torch_adam": true,
            "adam_w_mode": true
        }
    }
```
